routes/campaign.php

```diff
@@ -131,6 +131,7 @@
 Route::get('/calendars/{calendar}/month-list', 'CalendarController@monthList')->name('calendars.month-list');
 Route::get('/calendars/{calendar}/events', 'CalendarController@events')->name('calendars.events');
 Route::get('/calendars/{calendar}/today', 'CalendarController@today')->name('calendars.today');
+Route::get('/calendars/{calendar}/validate-length', [\App\Http\Controllers\CalendarController::class, 'eventLength'])->name('calendars.event.length');
```

Comment:
Please use `-` instead of `.`, as we use `.` for subfolders that end up as `/` in the url